replication-api-impl/pom.xml

```diff
@@ -31,6 +31,39 @@
             <groupId>org.springframework.data</groupId>
             <artifactId>spring-data-solr</artifactId>
             <version>${spring-data-solr.version}</version>
+            <exclusions>
+                <exclusion>
+                    <artifactId>solr-solrj</artifactId>
+                    <groupId>org.apache.solr</groupId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+        <dependency>
```

Comment:
This was taken from https://github.com/spring-projects/spring-data-solr/blob/4.1.0.RC2/pom.xml#L97-L123